content/cybersecurity/concepts/malware/terms/keylogger/keylogger.md

```diff
@@ -0,0 +1,55 @@
+---
+Title: 'Keylogger'
+Description: 'Keyloggers are a type of computer malware that records keystroke events on the keyboard and saves them to a log file, allowing it to steal sensitive data like passwords.'
+Subjects:
+  - 'Cybersecurity'
+  - 'Computer Science'
+Tags:
+  - 'Cybersecurity'
+  - 'Malware'
+CatalogContent:
+  - 'introduction-to-cybersecurity'
+  - 'paths/fundamentals-of-cybersecurity'
+---
+**Keyloggers** are a type of computer [Malware](https://www.codecademy.com/resources/docs/cybersecurity/malware) that records keystroke events on the keyboard and saves them to a log file, allowing them to steal sensitive data.
```

Comment:
```suggestion
**Keyloggers** are a type of computer malware that records keystroke events on the keyboard and saves them to a log file, allowing the theft of sensitive data.
```
Note, we generally don't link to parent entries because the Docs UI provides those links for each entry.